particles/Layout/Layout.recipes

```diff
@@ -1,5 +1,5 @@
 import 'Detail.recipes'
 import 'TabbedLayout.recipes'
-import 'AnimatedTiles.recipes'
-import 'BottomScrollTest.recipes'
+//import 'AnimatedTiles.recipes'
```

Comment:
Is this and next line intentional?

Reply:
Those are test/example recipes, I think they shouldn't be in `canonical.manifest`. I will remove the line entirely.